apps/teamsjs-cc-app/tabs/src/components/capabilities/Dialog.tsx

```diff
@@ -0,0 +1,53 @@
+import { app, dialog } from "@microsoft/teams-js";
```

Comment:
Would be good for this file and the next to have name changes to Dialog.**Url**.tsx and Dialog.*Url*.Bot.tsx to match what you've done with adaptive card dialogs